packages/atlas/src/views/studio/YppDashboard/tabs/YppDashboardMainTab.tsx

```diff
@@ -86,6 +87,15 @@ export const YppDashboardMainTab: FC = () => {
     <>
       <YppAuthorizationModal unSyncedChannels={unsyncedChannels} />
       <LayoutGrid>
+        <GridItem colSpan={{ xxs: 12, md: 12 }}>
```

Comment:
```suggestion
        <GridItem colSpan={{ base: 12 }}>
```
Base should do, otherwise LGTM